tests/lib/eslint/eslint.js

```diff
@@ -4603,6 +4608,22 @@ describe("ESLint", () => {
 
             await assert.rejects(() => eslint.isPathIgnored(null), /'filePath' must be a non-empty string/u);
         });
+
+        it("should warn if .eslintignore file is present", async () => {
```

Comment:
This is in `describe()` for the `ESLint#isPathIgnored()` method.

```
  ESLint
    isPathIgnored
      √ should warn if .eslintignore file is present
```

 Since we are testing a warning emitted from the constructor, I think we should move this into `describe("ESLint constructor function", () => { ... })`.

Reply:
Moved.